src/app/static/src/app/apiService.ts

```diff
@@ -46,15 +47,15 @@ export class APIService<S extends string, E extends string> implements API<S, E>
 
     static getFirstErrorFromFailure = (failure: Response) => {
         if (failure.errors.length == 0) {
-            return APIService.createError("API response failed but did not contain any error information. Please contact support.");
+            return APIService.createError("apiMissingError");
         }
         return failure.errors[0];
     };
 
     static createError(detail: string) {
         return {
             error: "MALFORMED_RESPONSE",
-            detail
+            detail: i18next.t(detail)
```

Comment:
So this of course means that once the error is displayed, if the language is toggled it remains in the language it was generated with. But this is the same situation as with the errors coming back from the API.

Reply:
I think that's fair enough!